cmd/codegen/gengo.go

```diff
@@ -38,6 +38,7 @@ var replace = map[string]string{
 	"igGetStyle":              "CurrentStyle",
 	"igGetMouseCursor":        "CurrentMouseCursor",
 	"ImAxis":                  "PlotAxisEnum",
+	"GetItemID":               "GetItemLabelID",
```

Comment:
I think `GetItemByID` was better there.

Reply:
yah, the line should be
```go
"GetItem_ID":               "ItemBylID",
```
because the name is missleading - itsuggest that the method returns Item's ID, but it actually returns Item basing on its ID :smile:

Reply:
"GetItem_ID" does not match any function name.

"ItemIDByLabel" sounds more correct:

``` c
typedef unsigned int        ImGuiID;// A unique ID used by widgets (typically the result of hashing a stack of string)

ImGuiID ImPlotItemGroup_GetItemID(ImPlotItemGroup *self, const char *label_id);
```

Reply:
renamed to

```
"GetItemID":               "ItemIDByLabel",
```

Reply:
it doesn't make sense imao, since
`func (self PlotItemGroup) ItemByLabelID(label_id string) ID {`
ItemByLabelID does **not** return Item, and `ItemID` does not return ID - it returns Item
so, imo i should be `ItemByID(ID) Item` and `ItemID(string) ID`

Reply:
>  so, imo i should be ItemByID(ID) Item and ItemID(string) ID
Have done as you suggested.